File: src/main/java/org/kite9/diagram/common/elements/mapping/IndependentCornerVertices.java
package org.kite9.diagram.common.elements.mapping;

import org.kite9.diagram.model.DiagramElement;

public class IndependentCornerVertices extends AbstractBaseCornerVertices {
		
	public IndependentCornerVertices(DiagramElement c, int depth) {
		super(c, FULL_RANGE, FULL_RANGE, depth);
		createInitialVertices(c);
	}


}

File: src/main/kotlin/org/kite9/diagram/common/elements/mapping/IndependentCornerVertices.kt
package org.kite9.diagram.common.elements.mapping

import org.kite9.diagram.model.DiagramElement

class IndependentCornerVertices(c: DiagramElement, depth: Int) :
    AbstractBaseCornerVertices(c, FULL_RANGE, FULL_RANGE, depth) {
    init {
        createInitialVertices(c)
    }
}